tests/seeds/archives-unit-files.sql

```
@@ -0,0 +1,145 @@
+     '3 fascicoli, 12 buste',
+     'Fondo di test per verifica caricamento documenti multipli per unità archivistica.',
+     'ita',
+     'text', 'E2EFILE', NOW());
+
+-- ── 3. Item A (figlio del fondo) ────────────────────────────────────────────
+INSERT IGNORE INTO archival_units
+    (parent_id, reference_code, institution_code, level, constructed_title,
+     date_start, date_end, scope_content, language_codes, specific_material,
+     local_classification, created_at)
+SELECT
+    (SELECT id FROM archival_units WHERE reference_code = 'E2E_FILE_FONDS_001' LIMIT 1),
```

SQLFluff (4.1.0)

[error] The 'WHERE' keyword should always start a new line.

(LT14)